opendatahub-io/data-science-pipelines-operator
# Rebase from dspv1 main, Add v2undeploy command to Makefile

## Description of your changes:
Rebase from `main` (pull in CI fixes)
Adds v2undeploy command to Makefile

## Testing instructions
`make v2deploy`, then `make v2undeploy` and ensure all dsp-v2 objects are cleaned up

## Checklist
